dateparser/date.py

```diff
@@ -248,10 +258,26 @@ def __init__(self, languages=None, allow_redetect_language=False):
             self.language_detector = AutoDetectLanguage(languages=None, allow_redetection=False)
 
     def get_date_data(self, date_string, date_formats=None):
-        """ Return a dictionary with a date object and a period.
-        Period values can be a 'day' (default), 'week', 'month', 'year'.
-        It aims to solve the following issue:
-        In example, a forum could displays "2 weeks ago" in the thread list
+        """ 
+        Parse string representing date and/or time in recognizeable localized formats.
+        Supports parsing multiple languages.
+
+        :param date_string:
+                A string representing date and/or time in a recognizably valid format.
+        :param date_formats:
+                A list of format strings using directives as given here_.
+                The parser applies formats one by one, taking into account the detected
+                languages.
+
+        .. _here: https://docs.python.org/2/library/datetime.html#strftime-and-strptime-behavior
+            :param date_formats:
+
+        :return: a dictionary with a :mod:`datetime.datetime` object and a *period*.
+        .. note:: *Period* values can be a 'day' (default), 'week', 'month', 'year'.
+
+        *Period* aims to solve the following issue:
+
+        For example, a forum could displays "2 weeks ago" in the thread list
```

Comment:
I think it's relevant more for bot_engines and not for dateparser. Though we need to explain period to the users more. In a sense it's a margin of error on how we calculate a data. May be give few more examples here (like for 'year'/'month' periods). Explain the uncertainty of "2 weeks ago" date.